infoschema/tables.go

```diff
@@ -561,6 +563,16 @@ var tableTiDBHotRegionsCols = []columnInfo{
 	{"FLOW_BYTES", mysql.TypeLonglong, 21, 0, nil, nil},
 }
 
+var tableAnalyzeStatusCols = []columnInfo{
+	{"TABLE_SCHEMA", mysql.TypeVarchar, 64, 0, nil, nil},
```

Comment:
`database_name` is more common on the user side.

Reply:
It keeps the convention with others in this file.